daily-coding-challlenges/21-05-2020/indexOf.js

```diff
@@ -7,4 +7,13 @@
 */
 const myIndexOf = (arr, searchVal) => {
 	/* Your implementation goes here */
-}
\ No newline at end of file
+	 let valIndex = 0;
+	for (let i = 0; i < arr.length; i++) {
+    if (arr[i] === searchVal) {
+		 return i;
+    } 
+  } 
+  return valIndex;
```

Comment:
@iamfortune, can you please check this again? 
The expectation is that the function returns -1 if the search value doesn't exist.